templates/taxi/manufacturer_list.html

```diff
@@ -3,48 +3,44 @@
 {% block content %}
   <h1>
     Manufacturer List
-    <a href="{% url 'taxi:manufacturer-create' %}" class="btn btn-primary link-to-page">
+    <a href="{% url 'taxi:manufacturer-create' %}" class="btn btn-primary ml-3">
       Create
     </a>
   </h1>
 
   {% if manufacturer_list %}
-    <table class="table">
-      <tr>
-        <th>ID</th>
-        <th>Name</th>
-        <th>Country</th>
-        <th>Update</th>
-        <th>Delete</th>
-      </tr>
-
-      {% for manufacturer in manufacturer_list %}
+    <table class="table table-bordered">
+      <thead>
         <tr>
-          <td>
-              {{ manufacturer.id }}
-          </td>
-          <td>
-              {{ manufacturer.name }}
-          </td>
-          <td>
-              {{ manufacturer.country }}
-          </td>
-          <td>
-              <a href="{% url 'taxi:manufacturer-update' pk=manufacturer.id %}">
+          <th>ID</th>
+          <th>Name</th>
+          <th>Country</th>
+          <th>Update</th>
+          <th>Delete</th>
+        </tr>
+      </thead>
+      <tbody>
+        {% for manufacturer in manufacturer_list %}
+          <tr>
+            <td>{{ manufacturer.id }}</td>
+            <td>{{ manufacturer.name }}</td>
+            <td>{{ manufacturer.country }}</td>
+            <td>
+              <a href="{% url 'taxi:manufacturer-update' pk=manufacturer.id %}" class="btn btn-warning btn-sm">
                 Update
               </a>
             </td>
             <td>
-              <a style="color: red"
-                href="{% url 'taxi:manufacturer-delete' pk=manufacturer.id %}">
+              <a href="{% url 'taxi:manufacturer-delete' pk=manufacturer.id %}" class="btn btn-danger btn-sm" style="color: white;">
                 Delete
               </a>
```

Comment:
Consider adding a `Cancel` button alongside the `Delete` button to meet the checklist requirement: "Ensure that there is a `Cancel` button alongside the `Delete` button in the templates."